### PR TITLE
CA-250235 - XC is allowing user to configure Health check even with W…

### PR DESCRIPTION
**Pull request by letsboogey**
…rong XenServer Credentials

### CODE REVIEW

File: XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs [hunk 1]
```diff
@@ -316,7 +319,6 @@ private void credentials_TextChanged(object sender, EventArgs e)
 
         private void xsCredentials_TextChanged(object sender, EventArgs e)
         {
-            UpdateButtons();
```
Comment (MihaelaStoica) [inline XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs]: By removing this line, the OK button stays disabled even after the XS credentials have been entered (but not checked with the "Test Ctedentials" button) 
Was this change intentional?

Comment (letsboogey) [inline XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs]: Yes this was intentional, to only allow the user to click ok after the credentials have been checked..since we couldn't do the check when the ok button is pressed

Comment (MihaelaStoica) [inline XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs]: But would the user know that they need to press "Test credentials" in order for the OK button to become enabled?
